packages/notification-center/src/hooks/useUnreadCount.ts

```diff
@@ -66,7 +66,7 @@ export const useUnreadCount = ({ onSuccess, ...restOptions }: UseQueryOptions<IC
     );
 
     return () => {
-      socket.off(WebSocketEventEnum.UNSEEN);
+      socket.off(WebSocketEventEnum.UNREAD);
```

Comment:
I think in line 78 we would need to update to: 
```
enabled: isSessionInitialized && fetchingStrategy.fetchUnreadCount
```

Reply:
Ok I'll add this

Reply:
@p-fernandez  I've updated the condition. But I needed to add a couple of things more because the fetchingStrategy interface was missing the fetchUnreadCount key.

For the default value, I set it to `true` so that existing code won't break (it was using unseen's default and it was true)